src/server/model/mod.rs

```diff
@@ -35,6 +40,20 @@ impl RadioContext {
     }
 }
 
+#[derive(Serialize, SimpleObject)]
```

Comment:
I'm also interested in additional summary like total topics covered, average message count (or some other things mentioned in the message propagation issues), but yeah some are not so easy to get from the existing code

It looks like the two HashMaps are indexed by indexer address, so having a Vec for active_indexers doesn't seem so useful when they can grab the keys from one of the maps.

I also think it might be easier to simply return the vec of IndexerStats (same thing as `AggregatedIndexerStats`?) so that users can do manipulation independently with greater flexibility

Reply:
> total topics covered 
you mean all distinct ipfs hashes that Listener Radio has received messages for?

> average message count
i don't quite get this

Reply:
> > total topics covered
> 
> you mean all distinct ipfs hashes that Listener Radio has received messages for?

yep 
 
> > average message count
> 
> i don't quite get this

you have the total_message_count for each indexer already, I'm interested in the average of that across all indexers. but, as I alluded to in the previous comment, it is something the client can do once they receive the vec of indexerStats

Reply:
i agree, let's leave it to the client

Reply:
i will add one for distinct ipfs hashes